codebase2/codebase-sqlite/U/Codebase/Sqlite/Operations.hs

```diff
@@ -1125,80 +1178,153 @@ data NamesByPath = NamesByPath
 namesByPath ::
   BranchHash ->
   -- | A relative namespace string, e.g. Just "base.List"
-  Maybe Text ->
+  PathSegments ->
```

Comment:
Most of the rest of this file (up till the `recursiveTermSearch`) is just adding code to re-prefix mount paths onto name searches which will now possibly occur within a dependency mount.